how-tos/redirect-the-user.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Redirecting Users"
+sidebarTitle: "Redirecting Users"
+---
+
+A seamless authentication experience for users is key to working with PropelAuth, which means we have provided various methods of redirecting 
```

Comment:
I believe we changed this intro a bit for the blog and the blog sets the stage a little better

Reply:
Actually, the rest of this seems like an older version of the blog as well (propelauth.com vs example.com)

Reply:
Fixed this